build/__tools__/run-tests.sh

```diff
@@ -78,13 +78,16 @@ if [ $? == 0 ] ; then
 fi
 
 SAVE_FILE=$(mktemp /tmp/pdo-test.XXXXXXXXX)
+ENCLAVE_DB_FILE=${PDO_HOME}/data/enclave_db.json
```

Comment:
enclave_db seems a bit a misleading name as this is about the services and will have to contain also information on the storage, not only enclave services?  (e.g., while every enclave service has a storage service co-located, i thought the opposite is not true and there might be stand-alone storage services which the client would have to know about?

Reply:
agreed. will change the name. I agree we need a similar service for the storage services as well. will make a separate PR for that

Reply:
How is the database created? If we expect to create this by hand then it should be a toml file. If not, then there needs to be a utility to manage the database (add, list, remove entries). Did I miss that utility?

Reply:
currently, any client that creates a new contract creates or updates the client's local database, if database path is provided as input  during test-request, test-contract or pdo-create, I have updated pdo-shell as well, will push it.